extensions/powershell/src/generators/nuspec.ts

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Host } from '@microsoft.azure/autorest-extension-base';
+import { Project } from '../project';
+
+export async function generateNuspec(service: Host, project: Project) {
+  // If the file is already there, don't write a new one.
+  if (await service.ReadFile(project.nuspec)) {
+    return;
+  }
+
+  const authorsOwners = project.azure ? 'Microsoft Corporation' : '';
```

Comment:
can we pull these values from configuration, and then failing that, default to (azure-value):(non-azure-value)?